### ISSUE TITLE
Ember-cli 0.2.6 complaining about missing template compilers dependency

### ISSUE THREAD

**Issue by robertodecurnex** [Gaurav0/ember-cli-jquery-ui#9]
Addon templates were detected, but there are no template compilers registered for `ember-cli-jquery-ui`. Please make sure your template precompiler (commonly `ember-cli-htmlbars`) is listed in `dependencies` (NOT `devDependencies`) in `ember-cli-jquery-ui`'s `package.json`.

Seeme like they now want addons to specify the template compiler as a dependency. It's an error, not even a warning, seems stupid to me but that seems to be the new rule.

May be adding it without specifying version can work (defaulting to the one configured at the ember dependencies level)

**Comment by Gaurav0** [comment]
Thanks Roberto. Can you link me to any discussion or explanation of whatever this new rule is?

**Comment by robertodecurnex** [comment]
That's all I have right now. I'll try to dig a little more later today. Basically they are now watching addons too. 

I'll come back with more detailed info (and a fix if possible).

**Comment by shunchu** [comment]
Hmm... I'm suffering from the same issue after having upgraded Ember-Cli to the latest version:

```
...
"ember-cli-jquery-ui": "0.0.16"
...
```

```
$ ember -v
version: 0.2.7
node: 0.12.4
npm: 2.11.0
```

Thoughts?

**Comment by Gaurav0** [comment]
This is fixed on master, not 0.0.16

**Comment by shunchu** [comment]
Ah! Thank you! Appreciate the response.
